tools/harvest_source_import/remote_ckan/lib.py

```diff
@@ -498,9 +498,6 @@ def get_package_from_data(self, data):
         if cfg.get('private_datasets', None) is not None:
             ret['private_datasets'] = cfg['private_datasets']
             
-        if cfg.get('default_groups', None) is not None:
```

Comment:
@adborden I remove this because it was causing an error in the test where we actually use default groups - setting these here might break other harvest types depending on which they use - we might want to consider only adding what we need as we go